### PR TITLE
Updating documentation for certificates

### PR DESCRIPTION
**Pull request by LeandraH**
### Short description

I had to update the certificates for my laptop and had some problems with it so I updated the documentation with what worked for me. If I missed something, please add it :)

### CODE REVIEW

File: native/docs/manual-builds.md [hunk 1]
```diff
@@ -29,24 +29,36 @@ found [here](build-configs.md).**
...
 ```bash
-cd ios && bundle exec fastlane certificates
+cd ios && fastlane match development (or whichever environment you need)
 ```
 
+- If you get an error that you can't clone the repo `app-credentials`, request access.
+
+- The password that you need to fill in twice is saved in Passbolt as Digitalfabrik Fastlane Match.
+
+- The Apple account that you need to sign in to is the one of the app team, the password is in Passbolt. This is also
```
Comment (ztefanie) [inline native/docs/manual-builds.md:50]: That is not correct, your personal account is linked to the organization. i never logged in with the app team account on my macbook

File: native/docs/manual-builds.md [hunk 2]
```diff
@@ -29,24 +29,36 @@ found [here](build-configs.md).**
...
+  the step where you will need the second factor.
+
+- You will need to add all the app_identifier you want to work with, in case of Integreat and its whitelabels, that would
+  be `de.integreat-app,app.aschaffenburg,de.malteapp.` If you fill in an invalid one, you get a list with the available ones.
```
Comment (ztefanie) [inline native/docs/manual-builds.md:54]: Just add a hint to the file in ~app-credentials-repository: https://github.com/digitalfabrik/app-credentials/blob/main/apple%20app%20identifier%20list.txt

File: native/docs/manual-builds.md [hunk 3]
```diff
@@ -29,24 +29,36 @@ found [here](build-configs.md).**
...
+
 ##### Trouble Shooting
 
 - [Installing certificates hangs on `Cloning remote git repo...`](troubleshooting.md#bundle-exec-fastlane-certificates-hangs-on-cloning-remote-git-repo)
 - Secrets containing `'`: Exporting as environment variable is possible as
   follows: `export SECRET='<prefix>'"'"'<suffix>'`.
-- 2FA prompt: Message @Max or @Steffen.
+- 2FA prompt: Message Steffi, Andy, or Leandra.
```
Comment (ztefanie) [inline native/docs/manual-builds.md:61]: This info should rather be in passbolt where the passwort is, than in the repostitory.

File: native/ios/fastlane/README.md [hunk 4]
```diff
@@ -1,41 +1,38 @@
-# fastlane documentation
+fastlane documentation
```
Comment (ztefanie) [inline native/ios/fastlane/README.md:1]: Why has this changed?

File: native/docs/manual-builds.md (hunk 1 above)
Comment (LeandraH) [reply native/docs/manual-builds.md:50]: Huh. I did. I don't think my personal Apple account is linked to the organization?

File: native/ios/fastlane/README.md (hunk 4 above)
Comment (LeandraH) [reply native/ios/fastlane/README.md:1]: I ran the command :)

File: native/docs/manual-builds.md (hunk 3 above)
Comment (LeandraH) [reply native/docs/manual-builds.md:61]: I'd add it in both places :)

File: native/docs/manual-builds.md (hunk 1 above)
Comment (LeandraH) [reply native/docs/manual-builds.md:50]: Can you add some text about how to link a personal Apple account to the organization?

File: native/docs/manual-builds.md (hunk 3 above)
Comment (ztefanie) [reply native/docs/manual-builds.md:61]: then we need to keep it updated in both places

File: native/docs/manual-builds.md (hunk 1 above)
Comment (ztefanie) [reply native/docs/manual-builds.md:50]: You need to be either a) admin of the organization or b) be provided with the right to access "certificates, identifiers & profiles".
I added you as a admin. See the attached screenshot from appstoreconnect 
leandra

**Comment by ztefanie** [comment]
I am wondering if we should change the app signing for development to "automatic code signing", so for local development there is no need to access the current profiles. As we rarely to never build live apps locally, I think this could help us prevent apple struggles. But i am not quite sure if this will work as expected/hoped.

File: native/docs/manual-builds.md (hunk 3 above)
Comment (LeandraH) [reply native/docs/manual-builds.md:61]: True

File: native/docs/manual-builds.md (hunk 1 above)
Comment (LeandraH) [reply native/docs/manual-builds.md:50]: Ah, cool, thanks!

Comment (LeandraH) [reply native/docs/manual-builds.md:50]: Oh, nice, without logging into the app-team account, we also don't need the second factor anymore 🥳

**Review by ztefanie — APPROVED** [review]
Thanks for updating the docs :+1: :100:

File: native/docs/manual-builds.md [hunk 5]
```diff
@@ -29,24 +29,35 @@ found [here](build-configs.md).**
...
 - [Install and setup Fastlane](../../docs/cicd.md#fastlane-setup).
 - Read about [Apple Certificates](apple-certifcates.md).
+- To set up the certificates:
+  - have access to the `app-credentials` repo
+  - have access to the Passbolt folder "Integreat App Publishing"
+  - your personal Apple account needs to be added to the Digitalfabrik's account either as an admin or with the right to
+    access "Certificates, Identifiers & Profiles".
 
-#### Certificates Setup
+If you need any of these permissions, ask for them in the #app-team channel on Mattermost.
```
[the remark this answers is not in the document]
Comment (LeandraH) [reply native/docs/manual-builds.md:38]: Oooops, thanks!

**Pull request merged**